src/content/for_developers/release-process/automated.en.md

```diff
@@ -0,0 +1,189 @@
+
+## Project dependencies
+
+The Submariner projects have a dependency hierarchy with respect to their Go libraries and Docker images.
+Therefore the releases for each individual project must be created in a specific order.
+
+The Go dependency hierarchy flow is as follows:
+
+`shipyard` <- `admiral` <- `[submariner, lighthouse]` <- `submariner-operator`
+
+Note that the `submariner` and `lighthouse` projects are siblings and thus do not depend on one another. Also the
+`submariner-operator` components expect that `lighthouse` and `submariner` are aligned on the same exact version.
+
+The Docker image dependency hierarchy flow is as follows:
+
+`subctl binary` <- `shipyard dapper base image` <- `[admiral, submariner, lighthouse, submariner-operator]`
+
+The Dapper base image that is provided by `shipyard` for building and E2E tests in all of the other projects pulls in the
+`subctl` binary.
+
+## Release versions
+
+The `vx.x.x` version format is used for the git projects while `x.x.x` is used for Docker images.
+
+The typical workflow is to first create release candidate(s) for testing before creating the final release. The suggested
+naming convention is to append `-rcN` to the final version, for example `v0.5.0-rc0`, `v0.5.0-rc1` etc.
+
+Sometimes you may want to create a specific project release for testing prior to creating a release candidate. In this
+case, the suggested naming convention is to append `-preN`.
+
+## Create Submariner product release
+
+The following sections outline the steps to be taken in order to create a full Submariner product release. As an example,
+we'll use version `v0.5.0`.
+
+The release process is mostly automated and is centered on a release YAML file which is constantly updated.
+On each step, once your pull request is reviewed and merged, the CI will create the required pull requests for
+other projects and they in turn would need to be reviewed and merged.
+
+### Step 1: Create release for the `shipyard` project
+
+Navigate to the [releases](https://github.com/submariner-io/releases) repository and fork it.
+
+1) Create a new branch for your intended release.
+
+2) Author a new release file in the releases directory (You can copy the example.yaml file).
+
+3) Fill in the general fields for the release.
+
+4) Create a link to the file to make it the target release (So CI knows what we're releasing):
+
+   ```bash
+   ln -s <your-release-file.yaml> target
+   ```
+
+5) Commit the new files and create a new pull request.
+
+### Step 2: Create release for the `admiral` project
+
+Once the Shipyard pull request to pin the Admiral project is merged, use its commit id to pin to Admiral.
+
+1) Create a new branch, or use your branch from before, on the fork of releases repository.
+
+2) Update the `status` field to `admiral`.
+
+3) Add the `admiral` component with the commit hash to the release yaml.
+
+4) Commit the modified file and create a new pull request.
```

Comment:
As before, what happens now? I assume CI runs to create the shipyard release? Do we merge the PR?